### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:  Jaws and NVDA  announces additional inappropriate information when "Push Image" dialog is opened 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1482]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Navigate to and select Project ->Azure->Push Image.
4. Push Image dialog will open.
5. Observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws  and NVDA announces additional inappropriate information when "Push Image" dialog is opened 

**Jaws behaviour:**
 Push Image dialog
Invalid Data: Please specify a valid docker file.Artifact Docker File Server URL Username Password Image and tag Container Registry <Server URL>/ Before launch: Build Artifacts, Activate tool window 
Combo box closed
blank
1 of 1
To change the selection use the Arrow keys.

**NVDA Behaviour:** 
Push Image  dialog  
Invalid Data: Please specify a valid docker file.
Artifact
Docker File
Server URL
Username
Password
Image and tag
Container Registry
 /
Before launch: Activate tool window
combo box  collapsed
combo box  collapsed
list
list item  selected  1 of 1

**Expected Result:**
 Jaws and NVDA should announce only the dialog name when dialog is opened .
i.e  Push Image dialog

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![push image-opened](https://user-images.githubusercontent.com/22956130/36105798-8a0a3eb6-103b-11e8-8624-9a5314245470.jpg)
![fb67aa83](https://user-images.githubusercontent.com/22956130/36105816-8da95854-103b-11e8-9fa4-6075a113b82b.jpg)

**Comment by Eskibear** [comment]
duplicated with #1485

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749909
